test/blackbox-tests/test-cases/too-many-parens/run.t

```diff
@@ -8,25 +8,39 @@ are readable.
   3:   (public_name hello)
   4:   (libraries (lib))
   5: ))
-  These parentheses are no longer necessary with dune, please remove them.
+  Error: Atom expected
+  Hint: try: in dune files, these parentheses are no longer necessary. Try removing them.
```

Comment:
The ` try:` feels out of place here

Reply:
Yes, it's hardcoded in `Hint`. I can move this.